src/bitmap/iter.rs

```diff
@@ -17,16 +17,16 @@ pub struct IntoIter {
 }
 
 impl Iter<'_> {
-    fn new(containers: &[Container]) -> Iter {
-        let size_hint = containers.iter().map(|c| c.len()).sum();
-        Iter { inner: containers.iter().flatten(), size_hint }
+    fn new(bitmap: &RoaringBitmap) -> Iter {
+        let size_hint = bitmap.len();
+        Iter { inner: bitmap.containers.iter().flatten(), size_hint }
```

Comment:
Can you explain to me the fact #179 removes the size_hint implementation and here we continue to track the length of the bitmap? I feel lost.